src/StructuredLogViewer.Core/NodeQueryMatcher.cs

```diff
@@ -43,7 +50,6 @@ public NodeQueryMatcher(string query, IEnumerable<string> stringTable)
                     return;
                 }
             }
-

```

Comment:
bring back the empty line - always an empty line after a `}` unless it's the last statement in the block